quic/s2n-quic-crypto/Cargo.toml

```diff
@@ -17,18 +17,13 @@ fips = ["aws-lc-rs/fips"]
 testing = []
 
 [dependencies]
+aws-lc-rs = { version = "1.9", features = ["prebuilt-nasm"] }
```

Comment:
I think we only want to enable this is our dev-dependencies? The problem with features is they are only additive and this would make it difficult for someone to opt out of this

Reply:
If its not on by default, wouldn't windows builds start breaking?

Could we add `prebuilt-nasm` as a default feature on `s2n-quic`, so customers can opt out by turning off default features?

Reply:
Yeah I guess this is part of the reason we haven't made the change. Well I guess today it's no worse than what windows environments get with ring... So let's move forward as-is and we can revisit the `default-features = false` route.